packages/components/src/Form/Fields/FieldCheckboxGroup/FieldCheckboxGroup.story.tsx

```diff
@@ -25,7 +25,11 @@
  */
 
 import React from 'react'
-import { FieldCheckboxGroup } from '@looker/components'
+import { Story } from '@storybook/react/types-6-0'
+import {
+  FieldCheckboxGroup,
+  FieldCheckboxGroupProps,
+} from './FieldCheckboxGroup'
 
 export default {
   title: 'Forms/CheckboxGroup',
```

Comment:
We've been flattening the navigation as we move stories over to `/packages/components`:
```suggestion
  title: 'CheckboxGroup',
```

Reply:
Noted! Resolved in c6191cb.